draft/authentication-basic.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: XXXX
+Author: Kyle Fuller
+Status: Draft
+Created: 2015-09-22
+Last Modified: 2015-09-28
+---
+
+# API Blueprint RFC XXXX: Basic Authentication Scheme
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Rationale](#rationale)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC adds the "Basic" authentication scheme for the Authentication
+framework proposed in RFC XXXX.
+
+## Motivation
+
+Basic authentication is a commonly used authentication mechanism, and is part
+of the HTTP/1.1 specification defined in
+[RFC1945 section 11.1](http://tools.ietf.org/html/rfc1945#section-11.1).
+
+## Rationale
+
+The "Basic" authentication scheme is based on the model that the user
+agent must authenticate itself with a user-ID and a password.
+
+As such, a basic authentication scheme may configure two properties,
+`username` and `password`. These properties indicate a sample username and
+password that may be used.
```

Comment:
What exactly are we configuring here? We say properties. But we are actually configuring sample username and sample password. I don't see any mention about the values being an example.

Also, I think we should have a default example username and example password for `Basic`.